### ISSUE TITLE
Incomplete - check console

### ISSUE THREAD

**Issue by runmd** [Esri/ago-assistant#270]
I get this error when copying a hosted feature service from ArcGIS Online to Portal for ArcGIS 10.6.1 using the Full Copy (Experimental).

**Comment by nixta** [comment]
Same here. Interestingly, the [service I'm copying](https://www.arcgis.com/home/item.html?id=5d02bf8a478d4732b90ec4b4f5ed89ee) has many feature layers. It fails on layers 4 and 8 (which have the bulk of the data) but other layers, including those with over 1000 records, do copy OK. I wonder if it's something to do with paging.

**Comment by slibby** [comment]
If you'd like to test further, check out the network tab for any failed requests being sent. Unfortunately we were never able to make this (experimental) feature as durable as hoped. We do not have any cycles to continue development on this application at this time. 

If it is any consolation, the ArcGIS Python API's `clone_items` method works VERY well for cloning hosted feature services

**Comment by nixta** [comment]
Yeah. They’re all CORS errors, which seems odd as I’d expect them _all_ to fail rather than only the ones with the beefier content. ¯\_(ツ)_/¯
